frontend/src/app/modules/interventions/map/map-switcher/map-switcher.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="switcher" [formGroup]="form">
+  <label class="switch-label">Pokaż interwencje: </label>
+  <mat-radio-group aria-labelledby="status-radio" formControlName="interventionStatus">
+    <mat-radio-button value="1">Do podjęcia</mat-radio-button>
```

Comment:
Let's reuse existing status options here.
In component.ts file, `import InterventionStatusOptions from intervention.status.ts`, add a public field like `statusOptions = IntervetnionStatusOptions`, 
then here in template
 ```
<mat-radio-button *ngFor="let option of statusOptions" [value]="option.value">{{option.displayValue}}</mat-radio-button>```

Reply:
Nice, much better! 👍